lib/clientv3/hostendpoint_e2e_test.go

```diff
@@ -146,9 +151,13 @@ var _ = testutils.E2eDatastoreDescribe("HostEndpoint tests", testutils.Datastore
 			outList, outError = c.HostEndpoints().List(ctx, options.ListOptions{})
 			Expect(outError).NotTo(HaveOccurred())
 			Expect(outList.Items).To(HaveLen(2))
-			testutils.ExpectResource(&outList.Items[0], apiv3.KindHostEndpoint, testutils.ExpectNoNamespace, name1, spec1)
-			testutils.ExpectResource(&outList.Items[1], apiv3.KindHostEndpoint, testutils.ExpectNoNamespace, name2, spec2)
-
+			if name1 < name2 {
```

Comment:
Seems like it would be much better to have an ExpectResourceList() assertion which asserts that the given resource is within the given list.

WDYT? This approach feels very brittle to me.

Reply:
Alternatively, we could just swap the order of the arguments in the Entry() so that name1 is always < name2?

That's still brittle though.

Reply:
It has snowballed a bit, but I think I've improved the situation by using ConsistOf and a gomega matcher....